code/game/atoms_movable.dm

```diff
@@ -4,6 +4,7 @@
 #if DM_VERSION >= 512
 #error HEY, PIXEL_SCALE probably exists now, remove this gross ass shim.
 #endif
+#endif

```

Comment:
oh god.... so what thats doing is embedding the rest of the entire project in the else if

meaning shit will break massively once we try to use 512